lib/Interpreter/LookupHelper.cpp

```diff
@@ -387,9 +387,9 @@ namespace cling {
                                 quickTypeName.size()-6);
       innerConst = true;
     }
-    constexpr int pointerType = 0;
-    constexpr int lrefType = 1;
-    constexpr int rrefType = 2;
+
+    enum PointerType { kPointerType, kLRefType, kRRefType, };
```

Comment:
How is this change related to noexcept/constexpr?  If it isn't, can we split it into its own commit?  Thanks.